resources/Spelunky2Entities.json

```diff
@@ -5001,20 +5005,17 @@
       { "field": "y_pos", "type": "Float" },
       { "field": "hand/feet_x_offset", "type": "Float" },
       { "field": "hand/feet_y_offset", "type": "Float" },
-      {
-        "field": "x_unk5",
-        "type": "Float",
-        "comment": "all of those are offsets/positions of different joints, i'm too lazy to document thme all"
-      },
-      { "field": "y_unk6", "type": "Float" },
-      { "field": "x_unk7", "type": "Float" },
-      { "field": "y_unk8", "type": "Float" },
-      { "field": "x_unk9", "type": "Float" },
-      { "field": "y_unk10", "type": "Float" },
+      { "field": "abs_x", "type": "Float" },
+      { "field": "abs_y", "type": "Float" },
+      { "field": "move_origin_x", "type": "Float" },
+      { "field": "move_origin_y", "type": "Float" },
+      { "field": "move_destination_x", "type": "Float" },
+      { "field": "move_destination_y", "type": "Float" },
       { "field": "unk11", "type": "Float" },
       { "field": "unk12", "type": "Float" },
       { "field": "unk13", "type": "Float" },
-      { "field": "move_timer", "type": "UnsignedWord" },
+      { "field": "elapsed_move_time", "type": "UnsignedByte" },
+      { "field": "move_timer", "type": "UnsignedByte" },
```

Comment:
This makes it inconsistent with the rest of the API where we tend to call the actual variables that count "timers" and specially for monsters: move timers, walk timers etc.

Reply:
Do you have a suggestion for the naming? The second var is how long the move will be, which is used as the divisor to calculate how much of the move has completed.

Reply:
`move_duration` ?